src/Symfony/Component/Console/Tests/Helper/ProgressBarTest.php

```diff
@@ -991,8 +1014,11 @@ public function testPreventRedrawFasterThan()
         rewind($output->getStream());
         $this->assertEquals(
             '    0 [>---------------------------]'.
+            $this->generateOutput('    1 [->--------------------------]').
             $this->generateOutput('    2 [-->-------------------------]').
-            $this->generateOutput('    4 [---->-----------------------]'),
+            $this->generateOutput('    3 [--->------------------------]').
+            $this->generateOutput('    4 [---->-----------------------]').
+            $this->generateOutput('    5 [----->----------------------]'),
```

Comment:
why do these tests need to change? same above

Reply:
It took me a while to notice the problem. @javiereguiluz, you flipped (min|max)SecondsBetweenRedraws. This test case tests throttling, it needs to use `minSecondsBetweenRedraws`. To avoid this kind of bugs in future, please use IDE rename feature.

Reply:
Sorry I forgot to update this PR. @ostrolucky thanks for telling me how to fix this problem. I did the changes you suggested and tests now pass. Thanks!